packages/server/graphql/private/mutations/connectSocket.ts

```diff
@@ -44,7 +44,7 @@ const connectSocket: MutationResolvers['connectSocket'] = async (
       .getAll(userId, {index: 'userId'})
       .filter({removedAt: null, inactive: true})('orgId')
       .run()
-    adjustUserCount(userId, orgIds, InvoiceItemType.UNPAUSE_USER, dataLoader).catch(console.log)
+    adjustUserCount(userId, orgIds, InvoiceItemType.UNPAUSE_USER).catch(console.log)
```

Comment:
+1 this one we will call most often, but that's still only once per session, so I think it's fine to remove the dataloader as you've did. I wouldn't like to do that as a general pattern though.

Reply:
I think in very few `update/insert` cases dataloader would safe us an additional db query, however we always have to remember to clear the cache. No strong opinion. Merging this for now, without taking this as a general pattern.